storage/src/lsm_tree/components/block_cache.rs

```diff
@@ -1,33 +1,85 @@
 use std::sync::Arc;
+use std::time::Instant;
 
 use bytes::BufMut;
 use futures::Future;
+use lazy_static::lazy_static;
 use moka::future::Cache;
+use prometheus;
 
 use super::Block;
 use crate::lsm_tree::DEFAULT_BLOCK_SIZE;
 use crate::{Error, Result};
 
+lazy_static! {
+    static ref BLOCK_CACHE_LATENCY_HISTOGRAM_VEC: prometheus::HistogramVec =
+        prometheus::register_histogram_vec!(
+            "block_cache_latency_histogram_vec",
+            "block_cache latency histogram vec",
+            &["op", "node"]
+        )
+        .unwrap();
+}
+
+struct BlockCacheMetrics {
+    block_cache_get_latency_histogram: prometheus::Histogram,
+    block_cache_insert_latency_histogram: prometheus::Histogram,
+    block_cache_fill_latency_histogram: prometheus::Histogram,
+}
+impl BlockCacheMetrics {
+    fn new(node: u64) -> Self {
+        Self {
+            block_cache_get_latency_histogram: BLOCK_CACHE_LATENCY_HISTOGRAM_VEC
+                .get_metric_with_label_values(&["block_cache_get", &node.to_string()])
+                .unwrap(),
+            block_cache_insert_latency_histogram: BLOCK_CACHE_LATENCY_HISTOGRAM_VEC
+                .get_metric_with_label_values(&["block_cache_insert", &node.to_string()])
+                .unwrap(),
+            block_cache_fill_latency_histogram: BLOCK_CACHE_LATENCY_HISTOGRAM_VEC
+                .get_metric_with_label_values(&["block_cache_fill", &node.to_string()])
+                .unwrap(),
+        }
+    }
+}
```

Comment:
I think we it's better to move the metrics into a separated file in `lsm_tree/components/metrics.rs` , name it `LsmTreeMetrics` and share the metrics object in `lsm_tree`. And we can remove the `node` arg in `BlockCache::new()`

Reply:
You can refer to how I did in `raft_log_store`.